### PR TITLE
[PL-49108] PL 1.34.x Release Notes

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
[PL-49108](https://harness.atlassian.net/browse/PL-49108)

https://harness.atlassian.net/wiki/spaces/PD/pages/21749825690/NG+Release+SAAS+April+15th+2024+for+1.34.x-+Build

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

[PL-49108]: https://harness.atlassian.net/browse/PL-49108?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: release-notes/platform.md [hunk 1]
```diff
@@ -80,16 +80,42 @@ The following deprecated API endpoints are longer supported:
...
+   
+   mode: Delegate
+   dryRun: false
+   workloadName: delegate-name
+   namespace: harness-delegate-ng
+   containerName: delegate
+   registryMirror: us.gsr.io/gcr-mirror
+   delegateConfig:
+   
+   ```
```
Comment (brian-f-harness) [inline release-notes/platform.md:109]:
```suggestion
- We've added an optional registry mirror configuration for delegate `upgrader`. If you use Docker pull through registry cache (`https://docs.docker.com/docker-hub/mirror/`), you can configure `upgrader` to use an optional registry mirror for your delegate images. For more information, go to [Configure an optional registry mirror for delegate images](/docs/platform/delegates/install-delegates/delegate-upgrades-and-expiration#configure-an-optional-registry-mirror-for-delegate-images). (PL-47920, ZD-59005)
```

File: release-notes/platform.md [hunk 2]
```diff
@@ -80,16 +80,42 @@ The following deprecated API endpoints are longer supported:
...
+
 - AWS Secret Manager connectors now support both plain text and encrypted type Access Key Ids. (PL-48420)
 
 - Docker delegate images are no longer pushed to `app.harness.io/registry`. To pull images, use `gcr.io/gcr-prod/harness/delegate:<IMAGE_TAG>`. This item requires Harness Delegate version 24.04.82707. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (PL-46947)
 
 #### Fixed issues
 
+- 2FA reset emails were failing to display the QR code properly due to the recent deprecation of Google APIs. The method for generating QR codes has been updated, resolving the issue and ensuring QR codes are now correctly included in 2FA reset emails. (PL-48980)
```
Comment (brian-f-harness) [inline release-notes/platform.md:117]:
```suggestion
- 2FA reset emails failed to display the QR code properly due to the recent deprecation of Google APIs. The method for generating QR codes has been updated, resolving the issue and ensuring QR codes are now correctly included in 2FA reset emails. (PL-48980, ZD-61314, ZD-61420, ZD-61486)
```

Comment (brian-f-harness) [reply release-notes/platform.md:117]: @saifeemustafaqharness please add the Zendesk ticket ID references.

File: release-notes/platform.md [hunk 3]
```diff
@@ -80,16 +80,42 @@ The following deprecated API endpoints are longer supported:
 
 ## April 2024
 
+### Version 1.34.x<!--  April 22, 2024 -->
+
+#### Fixed issues
+
+- The link to Slack on the UI Help section was not functioning. The community link has been updated to ensure it works correctly. (PL-48948)
```
Comment (brian-f-harness) [inline release-notes/platform.md:87]:
```suggestion
- The link to Slack on the UI Help section was not functioning. The community link has been updated to ensure it works correctly. (PL-48948, ZD-61203)
```

File: release-notes/platform.md [hunk 4]
```diff
@@ -80,16 +80,42 @@ The following deprecated API endpoints are longer supported:
...
+
+- When a new pipeline for approval was created, but the approval email was not being sent, preventing testing of the link within the approval email. Resolved this issue that prevented notifications from being sent to multiple emails configured in a userGroup for email notifications. (PL-48854)
```
Comment (brian-f-harness) [inline release-notes/platform.md:89]:
```suggestion
- When a new pipeline for approval was created, but the approval email was not being sent, preventing testing of the link within the approval email. Resolved this issue that prevented notifications from being sent to multiple emails configured in a user group for email notifications. (PL-48854, ZD-60366, ZD-61333)
```

File: release-notes/platform.md [hunk 5]
```diff
@@ -80,16 +80,42 @@ The following deprecated API endpoints are longer supported:
...
+
+- 2FA reset emails were failing to display the QR code properly due to the recent deprecation of Google APIs. The method for generating QR codes has been updated, resolving the issue and ensuring QR codes are now correctly included in 2FA reset emails. This item requires Harness Delegate version 24.03.82709. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate) (PL-48980, ZD-61314, ZD-61420, ZD-61486)
```
Comment (brian-f-harness) [inline release-notes/platform.md:91]:
```suggestion
- 2FA reset emails failed to display the QR code properly due to the recent deprecation of Google APIs. The method for generating QR codes has been updated, resolving the issue and ensuring QR codes are now correctly included in 2FA reset emails. This item requires Harness Delegate version 24.03.82709. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate) (PL-48980, ZD-61314, ZD-61420, ZD-61486)
```

**Review by brian-f-harness — CHANGES_REQUESTED** [review]

File: release-notes/platform.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 title: Platform release notes
 sidebar_label: Platform
 tags: [NextGen, "platform"]
-date: 2024-04-16:T10:00:30
+date: 2024-04-22:T10:00:30
```
Comment (brian-f-harness) [inline release-notes/platform.md:5]:
```suggestion
date: 2024-04-23:T10:00:30
```

File: release-notes/platform.md [hunk 7]
```diff
@@ -80,16 +80,42 @@ The following deprecated API endpoints are longer supported:
 
 ## April 2024
 
+### Version 1.34.x<!--  April 22, 2024 -->
```
Comment (brian-f-harness) [inline release-notes/platform.md:83]:
```suggestion
### Version 1.34.x<!--  April 23, 2024 -->
```

File: release-notes/platform.md (hunk 1 above)
Comment (brian-f-harness) [reply release-notes/platform.md:109]: @saifeemustafaqharness suggest a more concise RN here and link to the docs that are already in place. Also, please remember to add all applicable Zendesk ticket IDs for all applicable RN.

**Review by brian-f-harness — CHANGES_REQUESTED** [review]
Some suggested changes

File: release-notes/platform.md [hunk 8]
```diff
@@ -100,13 +114,13 @@ The following deprecated API endpoints are longer supported:
...
 
-- The delegate task rejection metric was designed to reflect tasks rejected by a delegate due to system-related reasons (such as lack of resources or exceeding the limit of parallel tasks) but did not include specific details like task type or task ID. Enhanced the task rejection metrics by adding `taskType` and `taskId` labels. (PL-48488)
+- The delegate task rejection metric was designed to reflect tasks rejected by a delegate due to system-related reasons (such as lack of resources or exceeding the limit of parallel tasks) but did not include specific details like task type or task ID. Enhanced the task rejection metrics by adding `taskType` and `taskId` labels. This item requires Harness Delegate version 24.04.82800. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate) (PL-48488)
```
Comment (brian-f-harness) [inline release-notes/platform.md:123]:
```suggestion
- The delegate task rejection metric was designed to reflect tasks rejected by a delegate due to system-related reasons (such as lack of resources or exceeding the limit of parallel tasks) but did not include specific details like task type or task ID. Enhanced the task rejection metrics by adding `taskType` and `taskId` labels. This item requires Harness Delegate version 24.04.82804. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate) (PL-48488)
```

File: release-notes/platform.md [hunk 9]
```diff
@@ -100,13 +114,13 @@ The following deprecated API endpoints are longer supported:
...
 
 - Users were being logged out when testing a Git connector with invalid credentials due to the Git client's 401 response being propagated to the UI. Implemented error handling to convert a 401 response from the test connection step to a 400, while preserving the original error message, preventing unintended user logouts. (PL-47753, ZD-58629)
```
Comment (brian-f-harness) [inline release-notes/platform.md:125]:
```suggestion
- Users were being logged out when testing a Git connector with invalid credentials due to the Git client's 401 response being propagated to the UI. Implemented error handling to convert a 401 response from the test connection step to a 400, while preserving the original error message, preventing unintended user logouts. This item requires Harness Delegate version 24.04.82804. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (PL-47753, ZD-58629)
```

File: release-notes/platform.md [hunk 10]
```diff
@@ -80,16 +80,30 @@ The following deprecated API endpoints are longer supported:
...
+
 - AWS Secret Manager connectors now support both plain text and encrypted type Access Key Ids. (PL-48420)
 
 - Docker delegate images are no longer pushed to `app.harness.io/registry`. To pull images, use `gcr.io/gcr-prod/harness/delegate:<IMAGE_TAG>`. This item requires Harness Delegate version 24.04.82707. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (PL-46947)
 
 #### Fixed issues
 
+- 2FA reset emails failed to display the QR code properly due to the recent deprecation of Google APIs. The method for generating QR codes has been updated, resolving the issue and ensuring QR codes are now correctly included in 2FA reset emails. (PL-48980, ZD-61314, ZD-61420, ZD-61486)
```
Comment (brian-f-harness) [inline release-notes/platform.md:105]:
```suggestion
- 2FA reset emails failed to display the QR code properly due to the recent deprecation of Google APIs. The method for generating QR codes has been updated, resolving the issue and ensuring QR codes are now correctly included in 2FA reset emails. This item requires Harness Delegate version 24.04.82804. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). (PL-48980, ZD-61314, ZD-61420, ZD-61486)
```

**Review by pratmit — APPROVED** [review]
LGTM

**Pull request merged**